### PR TITLE
Homework 5

### PR DESCRIPTION
**Pull request by Volodya227**

### CODE REVIEW

File: src/com/lits/homework5/IntegerArray.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        arr=new int[len];
+    }
+    int sum(){
+        var sum=0;
+        for(var i=0;i<arr.length;i++){
+            sum+=arr[i];
+        }
+        return sum;
+    }
+    Double e(){
```
Comment (ghost-lv) [inline src/com/lits/homework5/IntegerArray.java]: method name is not clear

File: src/com/lits/homework5/IntegerArray.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        var sum=0.0;
+        for(var i=0;i<arr.length;i++){
+            sum+=arr[i];
+        }
+        sum/=arr.length;
+        return sum;
```
Comment (ghost-lv) [inline src/com/lits/homework5/IntegerArray.java]: to use sum here is not ok

File: src/com/lits/homework5/CharArray.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
+package com.lits.homework5;
+
+public class CharArray extends Array {
+    CharArray(int len){
+        arr=new int[len];
+    }
+    void action(char c){
+        var s=0;
```
Comment (ghost-lv) [inline src/com/lits/homework5/CharArray.java]: better to use boolean

File: src/com/lits/homework5/CharArray.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+            if(((char)arr[i])==c&& s==0){
+                System.out.println(true);
+                s=1;
+            }
+        }
+    }
+    void printChar(char c){
+        var string="";
+        for(var i=0;i<arr.length;i++){
+            string=string+(char)arr[i];
```
Comment (ghost-lv) [inline src/com/lits/homework5/CharArray.java]: I'm not sure about result as charcode

File: src/com/company/Array.java [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+                for (var i = 0; i < arr.length; i++) {
+                    if (i % 2 == 1) {
+                        System.out.print(arr[i]+"  ");
+                    }
+                }
+            }
+
+        System.out.println("exit");
+    }
+    public void r(String anction,int number){
```
Comment (Volodymyr95) [inline src/com/company/Array.java]: not correct name

Comment (Volodymyr95) [inline src/com/company/Array.java]: what will be if I put "some action" on first argument?
incorrect method

**Pull request merged**